src/domain/payments/price-ratio.ts

```diff
@@ -16,21 +18,17 @@ export const PriceRatio = ({
   const convertFromUsd = (convert: UsdPaymentAmount): BtcPaymentAmount => {
     const amountAsNumber =
       (Number(convert.amount) * Number(btc.amount)) / Number(usd.amount)
-    const amount = BigInt(Math.round(amountAsNumber))
-    return {
-      amount: convert.amount === 0n ? 0n : amount === 0n ? 1n : amount,
-      currency: WalletCurrency.Btc,
-    }
+    const amount = Math.round(amountAsNumber)
+    const btcAmount = convert.amount === 0n ? 0n : amount === 0 ? 1n : amount
```

Comment:
2 `?` in one line => maybe the logic should be extract?

Reply:
Fixed in #1329